decidim-comments/app/queries/decidim/comments/comments_with_replies.rb

```diff
@@ -8,13 +8,15 @@ class CommentsWithReplies < Rectify::Query
       # Syntactic sugar to initialize the class and return the queried objects.
       #
       # commentable - a resource that can have comments
+      # options - Any additional options for this query.
```

Comment:
The options should be documented like this:

![](https://i.imgur.com/BKROorb.png)